src/components/Person.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {name}
```

Comment:
Краще записуй так, не варті 3 строки, щоб додати один пробіл ))
```suggestion
      {`My name is ${name}`}
```